### PR TITLE
Add coding practice section to Exploration

### PR DESCRIPTION
**Pull request by rwblickhan**
# Changes

Adds coding practice section to `Exploration`, containing Project Euler and LeetCode for now.

### CODE REVIEW

File: Exploration/README.md [hunk 1]
```diff
@@ -22,3 +22,14 @@ approved by Launch Pad tech leads!
   is an overview of (deep) neural networks, one of the most popular techniques
   for machine learning. It's written by Michael Nielsen, also known for one
   of the standard quantum computing textbooks.
+
+## Coding Practice
+
+* [LeetCode](https://leetcode.com) is probably the most popular website for coding
+  challenges and comes highly recommended for interview preparation - big companies
+  often take interview questions directly from LeetCode!
+* [Project Euler](https://projecteuler.net/) is a very different style of coding
```
Comment (bobheadxi) [inline Exploration/README.md]: we could also feature @iKevinY 's project here 😋 

https://github.com/iKevinY/EulerPy

Comment (iKevinY) [reply Exploration/README.md]: I'm in the process of fixing the Homebrew build right now! 😛

Comment (rwblickhan) [reply Exploration/README.md]: Ew no pipenv 😢

Comment (iKevinY) [reply Exploration/README.md]: @rwblickhan In my defence, I wrote it 5 years ago 😅

Comment (rwblickhan) [reply Exploration/README.md]: Not too late to change it 😛

**Pull request merged**